openlibrary/templates/work_search.html

```diff
@@ -202,14 +202,17 @@ <h1>$_("Search Books")</h1>
                 $ works = add_read_statuses(username, works)
             $ show_librarian_extras = ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/librarians'))
 
+            $# Subject key list will only be included in each doc if the patron has a sfw cookie
+            $ subject_keys = [d.get('subject', []) for d in docs]
             $ use_my_books_droppers = 'my_books_dropper' in ctx.features
-            $for work in works:
-                $if use_my_books_droppers:
+            $for work, subjects in zip(works, subject_keys):
+                $if not use_my_books_droppers:
```

Comment:
@jimchamp should this be `if use_my_books_droppers` then `dropper = None`? Or should it be `if **not** use_my_books_droppers`?

Reply:
```suggestion
                $if use_my_books_droppers:
```
Yeah, this is a little confusing...  Legacy droppers are passed into the SearchResultWork macro as `reading_log`.  If `include_dropper` is true, then we try to find the correct edition key in SearchResultWork and render the dropper there.

https://github.com/internetarchive/openlibrary/blob/eacde86bff6f3c4de7864b6af7729f480fe6d9fe/openlibrary/macros/SearchResultsWork.html#L129-L141

It would probably be better if `include_dropper` was instead named `render_dropper`.  `reading_log` should also be removed, and can be when we are comfortable with removing the legacy dropper code.